<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enter_value</class>
 <widget class="QDialog" name="enter_value">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Ink Free&quot;;
color: rgb(144, 0, 0);</string>
   </property>
   <property name="text">
    <string>Enter values to the Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Website or App Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="webText">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userText">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passText">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
